.github/workflows/release.yaml

```diff
@@ -0,0 +1,69 @@
+name: release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  package:
+    name: package and upload release assets
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - linux
+          - darwin
+          - windows
+        arch:
+          - amd64
+          - arm
+        exclude:
+          - os: darwin
+            arch: arm
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup-go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
+      - name: set package name
+        id: package-name
+        run: |
+          output_name=nebula-importer-${{ matrix.os }}-${{ matrix.arch }}-${GITHUB_REF#refs/tags/}
+          if [ "${{ matrix.os }}" = "windows"  ]; then
+              output_name+='.exe'
+          fi
+          echo "::set-output name=package_name::$output_name"
+      - name: build
+        env:
+          GOOS: ${{ matrix.os }}
+          GOARCH: ${{ matrix.arch }}
+        run: |
+          cd cmd
+          go build -o ../${{ steps.package-name.outputs.package_name }}
```

Comment:
The built binary of nebula-importer is a dynamic executable. Could specify ```CGO_ENABLED=0``` when use go build to make it a static binary
```
ldd ./nebula-importer
        linux-vdso.so.1 (0x00007ffd5b58f000)
        libpthread.so.0 => /usr/lib64/libpthread.so.0 (0x00007fe85a9d3000)
        libc.so.6 => /usr/lib64/libc.so.6 (0x00007fe85a80d000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fe85aa10000)
```

Reply:
Good Catch! Fixed.